.circleci/config.yml

```diff
@@ -375,7 +375,6 @@ jobs:
       - setup_remote_docker:
           docker_layer_caching: true
       - run: |
-          docker build .
```

Comment:
this whole job can actually go away (we get this from docker hub already), but we can do as a follow-up

Reply:
We can though we'd require to add it to the required check-in branch protection, right?
(I don't think it's the case)

Reply:
hmm... I thought I already did.... but yes, I can add it to branch protection rules right now (I'll only add buster since this PR)

Reply:
https://github.com/DataDog/dd-trace-py/pull/2517

Reply:
ordering doesn't matter, we can get these changes through first